static-site/components/nav/nav-logo.tsx

```diff
@@ -0,0 +1,43 @@
+"use client";
+
+import Image from "next/image";
+import { usePathname } from "next/navigation";
+import React from "react";
+
+import { groupsApp, siteTitle } from "../../data/BaseConfig";
+import logo from "../../static/logos/nextstrain-logo-small.png";
+
+export default function NavLogo(): React.ReactElement | null {
+  const pathname = usePathname();
+
+  return pathname !== "/" ? (
+    <>
+      <Logo />
+      <LogoType />
+    </>
+  ) : null;
+}
+
+const Logo = (): React.ReactElement => (
+  <a href="/" className="logo">
+    <Image src={logo} alt="Nextstrain (logo)" width="40" />
+  </a>
+);
+
+const LogoType = (): React.ReactElement => {
+  const rainbowTitle = siteTitle
+    .split("")
+    .map((letter, i) => (
+      <span key={i} style={{ color: "var(--titleColor" + i + ")" }}>{letter}</span>
```

Comment:
note to self: can you use variable interpolation here and make this string a little cleaner?

Reply:
yes, you can.